### PR TITLE
Update c++-quiz.md

### PR DESCRIPTION
**Pull request by Aditya-Dhodade**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [ ] I have made small correction/improvements

### Changes / Instructions

_Add instructions to me, please type here, thanks_

### CODE REVIEW

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
please fix merge conflicts, thanks

**Review by haxkd — CHANGES_REQUESTED** [review]

**Comment by Aditya-Dhodade** [comment]
Issue resolved and reverified the question and answer

File: c++/c++-quiz.md [hunk 1]
```diff
@@ -2524,8 +2524,8 @@ class my_class{
 
 ```cpp
 int matrix[3][3] = {{1, 2, 3},{4, 5, 6},{7, 8, 9}};
-for(int i=0;i<n;i++){
-   for(int j=0;j<n;j++){
+for(int i=0;i<3;i++){
+   for(int j=0;j<3;j++){
         int a = mat[i][j];
         mat[i][j] = mat[j][i];
```
Comment (haxkd) [inline c++/c++-quiz.md:2530]: is this is a linkedin assesment quiz??

**Comment by Aditya-Dhodade** [comment]
Yes, it is

**Comment by haxkd** [comment]
@Aditya-Dhodade resolve merge conflicts now

File: c++/c++-quiz.md (hunk 1 above)
Comment (Aditya-Dhodade) [reply c++/c++-quiz.md:2530]: yes

Comment (haxkd) [reply c++/c++-quiz.md:2530]: Resolve conflicts now

**Pull request closed without merging**